package.json

```diff
@@ -21,13 +21,12 @@
   },
   "homepage": "https://github.com/driftyco/ionic-module-template#readme",
   "devDependencies": {
```

Comment:
You might be right about the guides being for the main project. In any case, I don't have more time to throw after this problem right now.

We also you SASS; all our projects are complex. :)
And I'm aware that you can inline it, which is pretty cool for small components.

Thanks for working on this. We need a solid library build process eventually.